### PR TITLE
Preprocesing

### PR DESCRIPTION
**Pull request by MarceliG**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+    if args.download:
+        Manager.download_dataset(
+            author="maysee",
+            dataset_name="mushrooms-classification-common-genuss-images", 
+            save_loaction=Path.datasets
+        )
+
+    if args.train:
+        Train.execute(Path.muschroms, Path.models)
```
Comment (MarceliG) [inline main.py]:
```suggestion
        Train.execute(Path.mushrooms, Path.models)
```

File: src/train.py [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+        #     patience=5,
+        #     restore_best_weights=True
+        # )
+
+        # checkpoint = ModelCheckpoint(
+        #     save_path,
+        #     save_best_only=True,
+        #     monitor='val_loss',
+        #     mode='min'
+        # )
```
Comment (MarceliG) [inline src/train.py]: please uncoment, this will be helpfull

Comment (warzywko13) [reply src/train.py]: corrected

File: src/config/path.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import os
+
+class Path:
+    current_path = os.getcwd()
+    # directories
+    data_path = os.path.join(current_path, "data")
+    datasets = os.path.join(data_path, "datasets")
+    models = os.path.join(data_path, "models")
+    muschroms = os.path.join(datasets, "Mushrooms")
```
Comment (MarceliG) [inline src/config/path.py]:
```suggestion
    mushrooms = os.path.join(datasets, "Mushrooms")
```

**Pull request merged**